liquibase-core/src/main/java/liquibase/sqlgenerator/core/MarkChangeSetRanGenerator.java

```diff
@@ -113,6 +117,20 @@ public ValidationErrors validate(MarkChangeSetRanStatement statement, Database d
         }
     }
 
+    private Object column_Comments(ChangeSet changeSet) {
```

Comment:
The underscore in the function name is a bit different than our standard. Maybe more like `getCommentsColumn(changeSet)` or something?

Also, they are private functions and the target use right now is just Object, but still good to have the signature include the actual data type you expect.